policy/pa_test.go

```diff
@@ -326,10 +317,10 @@ func TestWillingToIssue_Wildcard(t *testing.T) {
 	test.AssertNotError(t, err, "Couldn't load policy contents from file")
 
 	domains := []string{
-		"perfectly-fine.com",             // fine
-		"letsdecrypt.org",                // banned
-		"ok.*.this.is.a.*.weird.one.com", // malformed
-		"also-perfectly-fine.com",        // fine
+		"perfectly-fine.com",      // fine
+		"letsdecrypt.org",         // banned
+		"example.com",             // banned
```

Comment:
Even with the PR description, it took me a sec to figure out what was going on here and why this was the right change. Maybe add a comment to the effect of "we don't test a malformed domain here, because doing so would cause WillingToIssue to error out before reaching the blocked names check".

Reply:
Yeah, it took me a few minutes to be sure the test failure was testing the right thing and that a change would be correct. I've updated this to test both the malformed + banned case and the banned case.